tests/data/parse/poetry_1_5/pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "aaa"
+version = "0.1.0"
+description = ""
+authors = ["Fridolin Pokorny <fridolin@redhat.com>"]
```

Comment:
@fridex wanna give this a new email?

Reply:
It's fine, at least spam bots will not reach out to me :)